civihr_employee_portal/js/ta-documents-app.js

```diff
@@ -124,7 +122,7 @@
            * there must be more that one contacts conidering at aleast a target contact in a document
            */
           $rootScope.$watch('cache.contact', function () {
-            availableContacts = $rootScope.cache.contact.arrSearch.length > 1;
+            availableContacts = $rootScope.cache.contact.arrSearch.length >= 1;
```

Comment:
> What happens if $rootScope.cache.contact.arrSearch.length === 0 ? Under which circumstances could it happen? It is empty during the data load, but is it possible if it empty after data load?

`$rootScope.cache.contact.arrSearch.length === 0` is not possible because there will be at least one contact (currently logged in user). It is empty during the data load because the currently logged in user has not been fetched and cached yet.